CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [2.2.0] - 2022-MM-DD
 ### Added
+
```

Comment:
I guess you meant to add a changelog.

Reply:
It had been added. There was a conflict with the master branch and I removed it temporarily.